server/mvcc/backend/backend.go

```diff
@@ -424,7 +433,7 @@ func (b *backend) defrag() error {
 		b.lg.Fatal("failed to rename tmp database", zap.Error(err))
 	}
 
-	b.db, err = bolt.Open(dbp, 0600, boltOpenOptions)
+	b.db, err = bolt.Open(dbp, 0600, &options)
```

Comment:
Isn't replacement of `boltOpenOptions` with `&options` an issue? I specifically mean setting `OpenFile` property 

https://github.com/etcd-io/etcd/blob/c46e96d519a38947188243c859e71ddd733e5065/server/mvcc/backend/backend.go#L384-L386

which was not present in `boltOpenOptions`